CMakeLists.txt

```diff
@@ -143,24 +136,6 @@ set(OPENCV_CMAKE_ARGS
 	-DBUILD_JPEG=OFF 
 	-DBUILD_LIST="core,ml,imgproc,highgui,flann,features2d,calib3d,video,objdetect,photo")
 
-if("${TARGET_PLATFORM}" MATCHES "win*")
```

Comment:
We still need all of these customizations to the OpenCV build, even as a submodule.